### ISSUE TITLE
Spelling correction

### ISSUE THREAD

**Issue by teusbenschop** [bibledit/bibledit#51]
Looking at the verse mappings page of bibledit-web it has the following text:

To update the verse mappings, make changes above, and save them.
Let's look at a sample mapping:
Genesis 31:55 = Genesis 32:1
It means that Genesis 31:55 in this verse mapping matches with Genesis 31:1 in the original Bible text. If this is the English verse mapping, it means that Genesis 31:55 in Bibles with the English versification system matches with Genesis 31:1 in Hebrew Bibles. The same applies to verse mappings for the New Testament. In short: At the left side of the equation sign is the passage in your Bible. And at the right side is the passage in the Hebrew or Greek Bible.

Should the text "Genesis 31:1” be 32:1?

The short version is actually clearer to me and might be better coming first.

**Comment by teusbenschop** [comment]
Fixed.
